packages/calcite-components/src/components/text-area/text-area.tsx

```diff
@@ -426,34 +430,50 @@ export class TextArea
     };
   }
 
-  private replacePlaceholdersInMessages(): string {
+  private replacePlaceholdersInLongMessages(): string {
     return this.messages.tooLong
-      .replace("{maxLength}", this.localizedCharacterLengthObj.maxLength)
-      .replace("{currentLength}", this.localizedCharacterLengthObj.currentLength);
+      .replace("{currentLength}", this.localizedCharacterLengthObj.currentLength)
+      .replace("{maxLength}", this.localizedCharacterLengthObj.maxLength);
   }
 
-  private isCharacterLimitExceeded(): boolean {
+  private replacePlaceholdersInShortMessages(): string {
```

Comment:
Can you extract a function that takes the templated string and reuse for both replacing both long and short messages?